### ISSUE TITLE
[BUG] EdXposed Framework is Installed, but not active

### ISSUE THREAD

**Issue by NAV-Management** [ElderDrivers/EdXposed#450]
**What happened?**
I upgraded Magisk to version 20.3 on my Xperia XZ1 Compact (Android 9, Security Patch 1. September 2019), and now it shows only the Message "EdXposed Framework is installed, but not active. Please check the logs for details"

So main functionality is for my device broken.

I updated to EdXpsoed Canary YAHFA V0.4.6.1 (4497) but it didn't helped. Before I updated EdXposed, I used the last Alpha v0.4.5.1.

In the Verbose Logs is the following message to find, which I think is the error: 
"java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference"

**Xposed/Xposed Module List**
Gravitybox [P] V.9.1.3
Lucky-P@tcher 8.6.3
XInternalSD V4.8
YouTube Deading, Lockscreen play V2.0.9

**Magisk/Magisk Module List**
Busybox for Android NDK V1.31.1
ExSDCard Access Enabler V3.5
FDE.AI V7.2
LKT V1.9.4
Magisk Manager for Recovery Mode (mm) V2019.4.4
Riru - Core V.19.6
Riru - EdXpsoed V0.4.6.1 (4497)-selhygwy (YAHFA)
Swift Installer Module V5
YouTube Vanced - Magisk Repo V14.21.54(b01.40.20)

**EdXposed Riru /Versions of EdXposed and Riru**

EdXposed: Riru - EdXpsoed V0.4.6.1 (4497)-selhygwy (YAHFA)

Riru: Riru - Core V.19.6

**Logcat/Logcat**

Verbose Logs are attached. Modules Logs has most same text, but not other text.

[all.log](https://github.com/ElderDrivers/EdXposed/files/4087751/all.log)

**Comment by drewcu** [comment]
Had similar problems myself with the framework when I experimented with Speed Mode and Dexopt within EdXposed Manager or when allowing FDE.AI to optimize my ART cache. My suggestion would be delete your cache/dalvik within TWRP and see if that fixes your issue. Then to not mess with your cache and possibly go back to EdXposed 0.4.5.1. That's what I did as Sandhook 4497 was uninstalling apps on its own after a reboot, YAHFA 4497 had too much lag, so now I am on YAHFA 4463 which doesn't seem to have the same lag as the new versions do.
